day2/src/main.rs

```diff
@@ -0,0 +1,7 @@
+use std::fs::read_to_string;
+
+fn main() {
+    let input = read_to_string("input.txt").unwrap();
```

Comment:
Could also have this do `include_str!` 🤷 I'd vote we either do:
1. CLI args to allow passing in a path and use `read_to_string`
2. static paths with `include_str!`

I lean towards 2) but 🤷

Reply:
282d5ee